sylius/mollie-plugin/2.0/config/routes/sylius_mollie.yaml

```
@@ -0,0 +1,2 @@
+sylius_mollie_plugin:
+    resource: "@SyliusMolliePlugin/config/routing.yaml"
```

```suggestion
    resource: "@SyliusMolliePlugin/config/routes.yaml"
```
I believe, as a rule of thumb, we use the `routes.yaml` name.